uptraining_docai_processor_using_python/docai_uptraining.ipynb

```diff
@@ -0,0 +1,995 @@
+{
```

Comment:
<p>Is this still needed with managed buckets?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/741/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
There are 2 options, user-managed and document ai managed. I've added a comment if user wants to go with document ai managed option